app/distributions/models.py

```diff
@@ -0,0 +1,40 @@
+from django.db import models
+from django.utils.translation import pgettext_lazy as _
+
+
+class Attribution(models.Model):
+
+    _context = "Attribution Model"
+
+    def __str__(self) -> str:
+        return str(self.name_en)
+
+    name_de = models.CharField(_(_context, "Name (German)"))
+    name_fr = models.CharField(_(_context, "Name (French)"))
+    name_en = models.CharField(_(_context, "Name (English)"))
+    name_it = models.CharField(_(_context, "Name (Italian)"), blank=True)
```

Comment:
I think we should allow the non-required fields to be empty on db-level as well, and only set fallback value on the fly when requested. Otherwise it will be difficult to track which values where actually provided in API calls.

Reply:
Not sure I understand what you mean.

The `blank=True` is only taken into account for form validation, not on what is written into the DB. You can see that for example in the admin page: The `blank=True` fields are shown in regular font, not bold as the required ones. So one can still write null or empty string (blank) to the DB if it's not via a form.

Probably I should also add the comment from [provider/models.py](https://github.com/geoadmin/service-control/blob/feat-PB-902-setup-models-v1-in-service-control/app/provider/models.py#L12) here:
```py
''''
Note: The "blank=False" for a model field doesn't prevent DB changes.
          It only has an effect on form validation.
''''
```

My impression is that this `blank=True` only creates confusion without a big gain (form validiation). So I propose to remove it.

Ok with you?

Reply:
Clarified in personal communication: `blank=True` allows empty fields in a form, `null=True` allows `null` values in the database. Often it's either `blank=True,null=True` or `blank=False,null=False`, but certain use cases can require only one of both `True`

Reply:
Should be adapted now, could you please have another look?